Misc/NEWS.d/next/Library/2021-12-29-14-42-09.bpo-43118.BoVi_5.rst

```diff
@@ -0,0 +1,2 @@
+In :func:`inspect.signature`, if the 'base' class has ``__text_signature__``
+attribute, use 'base' to help parse it. Patch by Weipeng Hong.
```

Comment:
This describes the implementation, but not the user-visible bug that it fixed. I'd suggest something like "Fix a bug in inspect.signature that was causing it to fail on some subclasses of classes with a __text_signature__ referencing module globals."